ui/app/AppLayouts/Profile/views/ChangePasswordView.qml

```diff
@@ -12,27 +12,82 @@ import StatusQ.Controls 0.1
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
 
 import AppLayouts.Profile.popups 1.0
 
 SettingsContentBase {
     id: root
 
     property var privacyStore
+
+    readonly property bool biometricsEnabled: localAccountSettings.storeToKeychainValue === Constants.keychain.storedValue.store
+    onBiometricsEnabledChanged: {
+        biometricsSwitch.checked = Qt.binding(() => { return root.biometricsEnabled });
+    }
+
     property var passwordStrengthScoreFunction: function () {}
 
-    //TODO https://github.com/status-im/status-desktop/issues/13302
-//    titleRowComponentLoader.sourceComponent: Item {
-//        implicitWidth: 226
-//        implicitHeight: 38
-//        StatusSwitch {
-//            LayoutMirroring.enabled: true
-//            text: qsTr("Enable biometrics")
-//            onToggled: {
-//                //
-//            }
-//        }
-//    }
+    titleRowComponentLoader.sourceComponent: Item {
+        implicitWidth: 226
+        implicitHeight: 38
+        StatusSwitch {
+            id: biometricsSwitch
+            LayoutMirroring.enabled: true
+            LayoutMirroring.childrenInherit: true
+            text: qsTr("Enable biometrics")
+            checked: root.biometricsEnabled
+            onToggled: {
+                enableBiometricsPopup.open();
+            }
+            StatusToolTip {
+                x: 15
+                visible: (!biometricsSwitch.checked && biometricsSwitch.hovered)
+                text: qsTr("Biometric login and transaction authentication")
+            }
+        }
+        StatusModal {
+            id: enableBiometricsPopup
+            title: biometricsSwitch.checked ? qsTr("Disable biometrics") : qsTr("Enable biometrics")
```

Comment:
I'd link it to the `root.biometricsEnabled` as it's supposed to be the single source of truth for this feature. Looks like the `biometricsSwitch.checked` can have intermediary states. Just to make sure we don't have false positive/negative here.

Same below